services/gitea/9.3.0/defaults/cm.yaml

```diff
@@ -46,9 +56,11 @@ data:
         periodSeconds: 10
         successThreshold: 1
         failureThreshold: 10
-    statefulset:
+    deployment:
       annotations:
         secret.reloader.stakater.com/reload: ${tlsCertificateSecret}
+      labels:
+        "deployment.kubernetes.io/pod-name": "gitea-0"
```

Comment:
These labels will now be added to every pod replica and the `-0` suffix isn't the clearest here (it was previously used to reference the `0` th index pod from the statefulset). 

Are there not labels that already exist on the pod from the gitea deployment spec (https://gitea.com/gitea/helm-chart/src/branch/main/templates/gitea/deployment.yaml#L40) that we could use?

Reply:
Apparently  I can re-use one if the labels:  `app.kubernetes.io/name=gitea`  
```shell
➜  k get po -l app.kubernetes.io/name=gitea -n kommander --kubeconfig test1xxx.conf
NAME                     READY   STATUS    RESTARTS   AGE
gitea-646459b978-tcgjl   1/1     Running   0          4h40m

```